packages/shared_preferences/shared_preferences_windows/example/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Regardless of the policy, these two license should have the same year (since either they should both be the set year, or both be current).